### PR TITLE
Remove click functionality on row entry

### PR DESCRIPTION
**Pull request by JuliaKirschenheuter**
Resolves https://github.com/nextcloud/server/issues/40672

Before:

Peek 2023-09-28 09-32

After:

Peek 2023-09-28 10-28

### CODE REVIEW

File: src/components/ActivityGrid.vue [hunk 1]
```diff
@@ -38,7 +38,7 @@
 			<tr v-for="(activity, activityKey) in group.activities" :key="activityKey">
 				<th scope="row">
 					<!-- eslint-disable vue/no-v-html -->
-					<label @click="toggleMethodsForActivity({groupKey, activityKey})" v-html="activity.desc" />
+					<label v-html="activity.desc" />
```
Comment (ShGKme) [inline src/components/ActivityGrid.vue:41]: This method was used only here, so we can also remove it from the component's methods (`mapActions`, in line 85).

But keeping the method in the store, if case we add a "check all" button later.

Comment (ShGKme) [reply src/components/ActivityGrid.vue:41]: Should we make this element just a `span`/`div`, as this `<label>` has no linked elements?

Comment (artonge) [reply src/components/ActivityGrid.vue:41]: Or shall we link it to the checkbox ?

Comment (ShGKme) [reply src/components/ActivityGrid.vue:41]: > Or shall we link it to the checkbox ?

We cannot, there are 2 checkboxes

**Review by ShGKme — APPROVED** [review]

**Review by Pytal — APPROVED** [review]

**Review by marcoambrosini — APPROVED** [review]

**Review by susnux — APPROVED** [review]

**Pull request merged**